src/index.html

```diff
@@ -1,14 +1,70 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <!-- <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    > -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="img">
+        <img src="./images/logo.png" alt="moyo-logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">smartphones</a>
+          </li>
```

Comment:
as those are on the same level of nesting and there are quite a lot of them - you can add some empty lines between them for better readability
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#samsung">samsung</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#smartphones">smartphones</a>
          </li>
```